testdata/README.md

```diff
@@ -2,6 +2,24 @@
 
 This directory contains cryptographic artifacts necessary for the Bootz implementation. The files here are included as examples only and real implementations would need their own method of generating, storing and retrieving these.
 
-## ca.pem
+## Regenerating test data
 
-This is an x509 certificate that represents the device manufacturer's root (or some intermediate) CA. It is required that the ownership voucher passed from the Bootz server is signed by this CA.
\ No newline at end of file
+All certificates, private keys and OVs can be regenerated by running the `generate` binary. Ensure all text files are deleted from this directory before running to avoid conflicts.
+
+The current files in this directory were generated using the command `./generate -vendor "Cisco" -owner "Google" -serials "123A,123B"`.
+
+### vendorca_{pub|priv}.pem
+
+This is an x509 certificate/RSA keypair that represents the device manufacturer's root (or some intermediate) CA. This is used to sign the OVs.
+
+### pdc_{pub|priv}.pem
+
+This is an x509 certificate/RSA keypair that represents the owner's Pinned Domain Cert.
+
+### oc_{priv|pub}.pem
+
+This is an x509 certificate/RSA keypair that represents the ownership certificate. It is signed by the PDC (or in some cases is the same as the PDC).
+
+### ov_x.txt
+
+Each of these files are base64-encoded, PKCS7 signed messages that represent the Ownership Voucher for that control card. It is signed by the Vendor CA and contains the PDC Cert.
```

Comment:
it might be good to explain what control card means here. Something like what we have in our internal `xx testing` page:

```
 These actions happen at the level of Control Card which is the smallest unit to host and run the OS image. FFF (Fixed Form Factor) devices only have one unit of this per chassis, while MFF (Modular Form Factor) devices often have two (or more) of these per chassis. The term Control Card isn't used consistently across the industry. Each vendor have their own hardware architectures and hence their own terms for this bootstrapping unit including:

- Arista uses supervisors e.g. DCS-7280 is a single-supervisor (FFF) and DCS-7808 is a dual-supervisor (MFF).
- Cisco uses Routing Processors (RPs) e.g. CISCO-8202 is a single-RP (FFF) and CISCO-8808 is a dual-RP (MFF).
- Juniper uses Routing Engine (REs) e.g. PTX1000 (AKA PTX1k) is a single-RE (FFF) and PTX10008 is a dual-RE (MFF).
```